include/ccl_core.h

```diff
@@ -82,6 +82,12 @@ typedef struct ccl_physical_constants {
   /**
    * T_ncdm, as taken from CLASS, explanatory.ini
    */
+  double T_ncdm;
+
+  /**
+   * TNCDM: repeat the value to preserve API because we renamed TNCDM to T_ncdm.
+   * TODO: Remove for CCLv3.
+   */
```

Comment:
Why are you renaming anything if you just want to move this to the cosmological parameters?

Reply:
Also, we should remove T_CMB from physical constants.

Reply:
For v3 we aim for consistency. It's T{underscore}CMB, so it would make sense to have T_ncdm instead of the hard to read `T_ncdm`. The latter is what CLASS uses too.